guidelines/enhancement_template.md

```diff
@@ -101,12 +101,55 @@ document.
 
 ### User Stories
 
-Detail the things that people will be able to do if this is implemented.
-Include as much detail as possible so that people can understand the "how" of
-the system. The goal here is to make this feel real for users without getting
-bogged down.
+Detail the things that people will be able to do if this is
+implemented. In each story, explain who the actor is based on their
+role, explain what they want to do with the system, and explain the
+underlying goal they have. Use the standard three part formula:
 
-Include a story on how this proposal will be operationalized:  lifecycled, monitored and remediated at scale.
+"As a <role>, I want to <take some action> so that I can <accomplish a
+goal>."
+
+Make the change feel real for users, without getting bogged down in
+implementation details.
+
+Include a story on how this proposal will be operationalized:
+lifecycled, monitored and remediated at scale.
+
+### Workflow Description
+
+Explain how the user will use the feature. Be detailed and explicit.
+Describe all of the actors, their roles, and the APIs or interfaces
+involved. Define a starting state and then list the steps that the
+user would need to go through to trigger the feature described in the
+enhancement. Optionally add a
+[mermaid](https://github.com/mermaid-js/mermaid#readme) sequence
+diagram.
+
+Use sub-sections to explain variations, such as for error handling,
+failure recovery, or alternative outcomes.
+
+For example:
+
+**cluster creator** is a human user responsible for deploying a
+cluster.
+
+**application administrator** is a human user responsible for
+deploying an application in a cluster.
+
+1. The cluster creator sits down at their keyboard...
```

Comment:
Can we make it a bit more realistic, "The cluster creator sits down at their keyboard" seems too detailed, imo. It should be clear, I agree but don't get into that much of a detail, and providing valid example is a starting point.  Example:

1. Cluster creator provides "some input" for the cluster...
2. ...
3. The cluster creator verifies the cluster is ready by looking at `oc get clusteroperator` status and all operators report `Ready`.

wdyt?

Reply:
I am +1 to @soltysh's comment.

Reply:
I've changed this step to talk about downloading the installer. 436ad6f4